### PR TITLE
RHIROS-1008 code refactoring to use generate API for systems pdf generation

### PR DESCRIPTION
**Pull request by PreetiW**
## Code refactoring to use generate API for systems pdf generation :boom:

Jira: [RHIROS-1008](https://issues.redhat.com/browse/RHIROS-1008)

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation requires update? :memo:

- [ ] Yes
- [X] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [X] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

**Review by kgaikwad — COMMENTED** [review]
@PreetiW  - could please add reference to backend change on which this PR is blocked?

File: src/Components/Reports/Common/SystemsFirstPage.js [hunk 1]
```diff
@@ -1,22 +0,0 @@
-import React, { Fragment } from 'react';
-import propTypes from 'prop-types';
-import { Section, Column } from '@redhat-cloud-services/frontend-components-pdf-generator';
```
Comment (r14chandra) [inline src/Components/Reports/Common/SystemsFirstPage.js:3]: As we are no longer using these packages -`@react-pdf/renderer` and 
`@redhat-cloud-services/frontend-components-pdf-generator`, we can remove these from the package.json file.

**Review by r14chandra — CHANGES_REQUESTED** [review]
The PR looks good 👍🏻 
Added two inline comments.

File: src/Components/Reports/Common/SystemsFirstPage.js (hunk 1 above)
Comment (PreetiW) [reply src/Components/Reports/Common/SystemsFirstPage.js:3]: nice catch @r14chandra, removed and pushed the changes. 💯

**Review by r14chandra — APPROVED** [review]

**Pull request merged**